CHANGELOG.md

```diff
@@ -3,7 +3,30 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.1.0 - 2021/07/28
+## v2.1.1 - 2021/11/23
```

Comment:
```suggestion
## v2.1.1 - 2021/11/25
```

Reply:
Thanks! Indeed, I was a little optimistic with the date.